Common-Issues.md

```diff
@@ -75,7 +75,7 @@ Under `repositories` in your `pom.xml`, you need to add a new `repository` for t
     ...
     <repository>
         <id>ess-repo</id>
-        <url>https://ci.ender.zone/plugin/repository/everything/</url>
+        <url>https://repo.essentialsx.net/releases/</url>
```

Comment:
This section should also mention the repo for snapshots.

Reply:
Might be worth adding something to clarify one or the other is needed, as most people probably don't need both. Could include the wording from the [README](https://github.com/EssentialsX/Essentials/#using-essentialsx-in-your-plugin).